### PR TITLE
SECURITY.md: Add key link

### PR DESCRIPTION
**Pull request by miri64**
### Contribution description
Adds a link to download the key to the SECURITY.md

### Testing procedure
- Read
- Check if Markdown is rendered correctly
- Check if link works

### Issues/PRs references
Follow-up on #17189

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -9,8 +9,11 @@ If a security issue is discovered, please report it to security@riot-os.org.
 A response will be provided within one week.
 The issue will be tracked in the [security mailing list](mailto:security@riot-os.org).
 The original reporter will be included in the discussion of the issue.
-You can encrypt your report using gpg key id 44C6AE441172F88D3423E81F5F7964D0F4239033,
-also included at the bottom of this file.
+You can encrypt your report using gpg key id
+[44C6AE441172F88D3423E81F5F7964D0F4239033][security-gpg], also included at the
```
Comment (kaspar030) [inline SECURITY.md:13]: this is not rendered correctly. Should it be `[44C6AE441172F88D3423E81F5F7964D0F4239033](security)`?

File: SECURITY.md [hunk 2]
```diff
@@ -9,8 +9,11 @@ If a security issue is discovered, please report it to security@riot-os.org.
...
+bottom of this file.
+
+[security]: https://riot-os.org/assets/keys/security.asc
```
Comment (kaspar030) [inline SECURITY.md:16]: and this, shouldn't it be `security-gpg`?

**Comment by kaspar030** [comment]
Please feel free to squash directly!

**Comment by miri64** [comment]
> Please feel free to squash directly!

Done

**Review by kaspar030 — APPROVED** [review]
ACK.

**Pull request merged**